docs/guides/client-vs-server-rendering.md

```diff
@@ -4,7 +4,7 @@
 
 In most cases, you should use the `prerender: false` (default behavior) with the provided helper method to render the React component from your Rails views. In some cases, such as when SEO is vital, or many users will not have JavaScript enabled, you can enable server-rendering by passing `prerender: true` to your helper, or you can simply change the default in `config/initializers/react_on_rails`.
 
-Now the server will interpret your JavaScript. The default is to use [ExecJS](https://github.com/rails/execjs) and pass the resulting HTML to the client. We recommend using [mini_racer](https://github.com/discourse/mini_racer) as ExecJS's runtime.
+Now the server will interpret your JavaScript. The default is to use Node.js.
```

Comment:
Think ExecJS will still be used behind the scenes but with Node.js rather than `mini_racer`. Maybe change this to reflect this and tell people they've got choice of alternative runtimes, as documented in ExecJS readme?

Reply:
That would be my suggestion too.

Reply:
Actually, can anything other than ExecJS be used? If not, the correct phrasing could be something like "Now the server will interpret your JavaScript using ExecJS. You can pick the engine as described in the documentation." If yes, this is a good place to explain how.